frontend/ui/widget/inputtext.lua

```diff
@@ -824,37 +821,29 @@ end
 -- offset is the absolute position, otherwise the offset is added to the current
 -- cursor position (negative offsets are allowed).
 function InputText:getChar(offset, is_absolute)
-    local idx
-    if is_absolute then
-        idx = offset
-    else
-        idx = self.charpos + offset
-    end
-    if idx < 1 or idx > #self.charlist then return end
+    local idx = is_absolute and offset or self.charpos + offset
     return self.charlist[idx]
 end
 
 function InputText:addChars(chars)
-    if not chars then
-        -- VirtualKeyboard:addChar(key) gave us 'nil' once (?!)
-        -- which would crash table.concat()
-        return
-    end
```

Comment:
Do your changes make sure this can't happen any longer?
Or are you taking a chance that it happened to me once but will never again ? :)

Reply:
I do not do `table.concat(chars)` anymore here.
I do `util.splitToChars(chars)` that accepts `nil` well.